src/scripts/start-stop-status

```diff
@@ -31,7 +31,7 @@ fi
 start_daemon() {
     local ts=$(date --iso-8601=second)
     echo "${ts} Starting ${SERVICE_NAME} with: ${SERVICE_COMMAND}" >${LOG_FILE}
-    STATE_DIRECTORY=${PKGVAR} ${SERVICE_COMMAND} >>${LOG_FILE} 2>&1 &
+    STATE_DIRECTORY=${PKGVAR} ${SERVICE_COMMAND} 2>&1 | sed -u '1,200p;d' >>${LOG_FILE} &
```

Comment:
maybe we should make line 201 say it's explicitly truncated, so people don't think it's hung or something?

```
$ (echo one; echo two; echo three; echo four; echo five) | sed '1,3 p; 4 s,.*,[further tailscaled logs suppressed], p; d'
one
two
three
[further tailscaled logs suppressed]
```

Reply:
Done.